app/src/docs/components/Search/searchUtils.js

```diff
@@ -95,12 +95,12 @@ export const runSearchQuery = ({ query, index, store }: any) => {
             q.term(lunr.tokenizer(query), {
                 fields: ['content'],
                 boost: 100,
-                usePipeline: false,
+                usePipeline: true,
```

Comment:
Unrelated to the search indexer but should be set `true` to improve search results.